tycho-extras/tycho-pomless/src/main/java/org/eclipse/tycho/pomless/TychoBundleMapping.java

```diff
@@ -170,11 +170,17 @@ private String getRequiredHeaderValue(String headerKey, Attributes headers, File
 
     private boolean isTestBundle(String bundleSymbolicName, File bundleRoot) throws IOException {
         Properties buildProperties = getBuildProperties(bundleRoot);
-        String property = buildProperties.getProperty("tycho.pomless.testbundle");
+        String property = buildProperties.getProperty(TYCHO_POMLESS_TESTBUNDLE_PROPERTY);
```

Comment:
As above, we should simply drop this code path and document it in the migration guide. It should be easy to perform a search+replace for anyone using it...

Reply:
Done.